onnxruntime/core/providers/cpu/sequence/sequence_ops.h

```diff
@@ -53,4 +53,22 @@ class SequenceConstruct final : public OpKernel {
   }
   Status Compute(OpKernelContext* context) const override;
 };
+
+class SplitToSequence final : public OpKernel {
```

Comment:
Tried to derive from SplitBase but it wasn't worth it. This op has several new variations compared to the original Split.